src/Tribe/Organizer.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+class Tribe__Events__Organizer {
+	const POSTTYPE = 'tribe_organizer';
+
+	/**
+	 * Args for organizer post type
+	 * @var array
+	 */
+	public $post_type_args = array(
+		'public'              => false,
+		'rewrite'             => array( 'slug' => 'organizer', 'with_front' => false ),
+		'show_ui'             => true,
+		'show_in_menu'        => 0,
+		'supports'            => array( 'title', 'editor' ),
+		'capability_type'     => array( 'tribe_organizer', 'tribe_organizers' ),
+		'map_meta_cap'        => true,
+		'exclude_from_search' => true,
+	);
+
+	public static $valid_keys = array(
+		'Organizer',
+		'Phone',
+		'Email',
+		'Website',
+	);
+
+	public $singular_organizer_label;
+	public $plural_organizer_label;
+
+	protected static $instance;
+
+	/**
+	 * Returns a singleton of this class
+	 *
+	 * @return Tribe__Events__Linked_Posts
+	 */
+	public static function instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self;
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor!
+	 */
+	public function __construct() {
```

Comment:
As with the matching class for venues should we make this protected or private, or mark as `@internal`?